lang/fr.js

```diff
@@ -68,7 +68,7 @@ export default {
 	"components.input-time-range.startTime": "Heure de début",
 	"components.interactive.instructions": "Appuyez sur Entrée pour interagir, sur Échapper pour quitter",
 	"components.link.open-in-new-window": "Ouvre une nouvelle fenêtre.",
-	"components.list.keyboard": "Use <b>arrow keys</b> to move focus inside this list, or <b>page up/down</b> to move up or down by 5",
+	"components.list.keyboard": "Utiliser les <b>touches fléchées</b> pour parcourir cette liste ou les touches <b>page up/down</b> pour monter ou descendre 5 éléments à la fois.",
```

Comment:
Ditto.

Reply:
Ha, this one says "5 elements", whereas `fr-fr` says 5cm 😖